cea/inputlocator.py

```diff
@@ -66,6 +66,11 @@ def get_optimization_slave_pp_activation_pattern(self, configkey):
         return os.path.join(self.get_optimization_slave_results_folder(),
                             '%(configkey)s_PPActivationPattern.csv' % locals())
 
+    def get_optimization_slave_pp_activation_cooling_pattern(self, configkey):
```

Comment:
Interesting question. Actually even I am not sure. I put it that way to be consistent with the other files being saved.